system/ee/legacy/libraries/Template.php

```diff
@@ -1454,7 +1463,8 @@ public function process_tags()
 
         for ($i = 0, $ctd = count($this->tag_data); $i < $ctd; $i++) {
             // Check the tag cache file
-            $cache_contents = $this->fetch_cache_file($this->tag_data[$i]['cfile'], 'tag', $this->tag_data[$i]['cache']);
+            $this->cache_prefix = (isset($this->tag_data[$i]['params']['cache_prefix'])) ? $this->tag_data[$i]['params']['cache_prefix'] : '';
+            $cache_contents = $this->fetch_cache_file($this->tag_data[$i]['cfile'], 'tag', $this->tag_data[$i]['params']);
```

Comment:
Is this correct? Or should it be:
`$cache_contents = $this->fetch_cache_file($this->tag_data[$i]['cfile'], 'tag', $this->tag_data[$i]['cache']);`

Reply:
No, that is intentional change. `cache` was holding params initially, but that's really mistake, because looking that code executed later on, it holds that caching status

Reply:
Okay - I wasnt sure based on just reading the code. It seemed like it could have been a copy paste error, or it could have been correct so I figured I'd ask first.